pkg/interface/soto/src/css/custom.css

```diff
@@ -0,0 +1,90 @@
+p, h1, h2, h3, h4, h5, h6, a, input, textarea, button {
```

Comment:
Does indigo just not allow for the kind of styles you want here or something? Was this flagged towards indigo devs? My understanding is we want to use it fully when we can, and make sure we can do that as often as possible.